### PR TITLE
Update to 1.8.0

### PR DESCRIPTION
**Pull request by cbouss**
Changes:
- update mypy to 1.8.0
- enable mypc

https://github.com/python/mypy/tree/v1.8.0

Note: remove temp channels before merging.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,55 +1,86 @@
...
+        - mypy_extensions >=1.0.0
+        - tomli >=1.1.0  # [py<311]
+        - types-psutil
+        - types-setuptools
+      run:
+        - python
+        - typing-extensions >=4.1.0
+        - mypy_extensions >=1.0.0
+        - tomli >=1.1.0  # [py<311]
+        # extra: dmypy
```
Comment (Jrice1317) [inline recipe/meta.yaml:46]: Is this syntax new?

Comment (cbouss) [reply recipe/meta.yaml:46]: No, just a regular comment :) 
It's an optional requirement we put in run.

File: abs.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+aggregate_branch: 3.12
+
+channels:
```
Comment (ryanskeith) [inline abs.yaml]: These two are now deployed as of last Friday.  So we shouldn't need the channels here anymore.

**Review by ryanskeith — APPROVED** [review]
LGTM, you could take out staging channels if you care too.  Though I don't think it's strictly necesary.

File: abs.yaml (hunk 2 above)
Comment (cbouss) [reply abs.yaml]: removed

**Review by SC426 — APPROVED** [review]

**Pull request merged**